.github/workflows/python-linters.yml

```diff
@@ -0,0 +1,46 @@
+name: Code quality
+
+on:
+  push:
+  pull_request:
+  schedule:
+  # Run every Friday at 18:02 UTC
+  # https://crontab.guru/#2_18_*_*_5
+  - cron: 2 18 * * 5
+
+jobs:
+  linters:
+    name: 🤖
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # max-parallel: 5
+      matrix:
+        os:
+        - ubuntu-18.04
+        env:
+        - TOXENV: docs
+        - TOXENV: lint
+        - TOXENV: lint-py2
+          PYTHON_VERSION: 2.7
+        - TOXENV: mypy
+        - TOXENV: packaging
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python ${{ matrix.env.PYTHON_VERSION || 3.7 }}
+      uses: actions/setup-python@v1
+      with:
+        version: ${{ matrix.env.PYTHON_VERSION || 3.7 }}
+    - name: Pre-configure global Git settings
+      run: >-
+        tools/travis/setup.sh
+    - name: Update setuptools and tox dependencies
+      run: >-
+        tools/travis/install.sh
+    - name: 'Initialize tox envs: ${{ matrix.env.TOXENV }}'
+      run: >-
+        TOX_PARALLEL_NO_SPINNER=1 python -m tox --parallel auto --notest --skip-missing-interpreters false
```

Comment:
```suggestion
        python -m tox --notest --skip-missing-interpreters false
```

Reply:
I mean, I probably wanted to keep it more generic. But it this is what you want, I'll patch it.